### PR TITLE
fix(iceworks-material-helper): generator page by blocks

### PR DESCRIPTION
**Pull request by caiyongmin**
Ref: issue https://github.com/ice-lab/iceworks/issues/754

### CODE REVIEW

File: extensions/iceworks-material-helper/web/src/pages/PageGenerator/index.tsx [hunk 1]
```diff
@@ -65,21 +70,24 @@ const Home = () => {
     }
   }
 
-  function generateBlockName(defineName: string): Promise<string> {
-    function generateName(setName, count = 0) {
-      const newName = !count ? setName : `${setName}${count}`;
-      const isConflict = selectedBlocks.some(({ name }) => name === newName);
+  // the string value will increment by 1
+  function generateBlockStringVal(defineValue: string, defineKey: string): string {
```
Comment (luhc228) [inline extensions/iceworks-material-helper/web/src/pages/PageGenerator/index.tsx:74]: 感觉 defineValue  defineKey 命名不是很恰当，originValue, originKey 会不会好点？

File: extensions/iceworks-material-helper/web/src/pages/PageGenerator/index.tsx [hunk 2]
```diff
@@ -65,21 +70,24 @@ const Home = () => {
...
+    function generateValue(setValue: string, setKey: string, count = 0) {
```
Comment (luhc228) [inline extensions/iceworks-material-helper/web/src/pages/PageGenerator/index.tsx:75]: setValue, setKey -> value, key ?

**Review by luhc228 — COMMENTED** [review]
reviewed

File: extensions/iceworks-material-helper/web/src/pages/PageGenerator/index.tsx (hunk 1 above)
Comment (caiyongmin) [reply extensions/iceworks-material-helper/web/src/pages/PageGenerator/index.tsx:74]: 好，这里我改一下

File: extensions/iceworks-material-helper/web/src/pages/PageGenerator/index.tsx (hunk 2 above)
Comment (caiyongmin) [reply extensions/iceworks-material-helper/web/src/pages/PageGenerator/index.tsx:75]: 👌，这里我改一下

**Review by luhc228 — APPROVED** [review]

**Pull request merged**